database/seedGenerator.js

```diff
@@ -0,0 +1,135 @@
+const faker = require('faker');
+// const fs = require('fs');
+const model = require('./db.js');
+
+const USERS_HEADERS = "userId,userName";
+const PROJECTS_HEADERS = "projectId,projectName,ownerId";
+const UPDATES_HEADERS = "id,title,postedBy,projectId,body,likes,pubDate,createdAt,updatedAt";
+const LIKES_LIMIT = 500;
+const BODY_PARAGRAPH_MAX = 15;
+const BODY_PARAGRAPH_MIN = 2;
+const MIN_UPDATES = 0;
+const MAX_UPDATES = 10;
+const KICKSTARTER_FOUNDED = new Date(2009, 3, 28);
+
+let updateCount = 0;
+
+function getName() {
+  return faker.name.findName();
+}
+
+function getLikes() {
+  return Math.floor(Math.random() * LIKES_LIMIT);
+}
+
+function getUpdateBody() {
+  // from 2 to 15 paragraphs
+  const paragraphCount =
+    Math.floor(Math.random() * BODY_PARAGRAPH_MAX - BODY_PARAGRAPH_MIN) + BODY_PARAGRAPH_MIN;
```

Comment:
I think you're missing a open paren before BODY_PARAGRAPH_MAX and an additional closing paren after the first BODY_PARAGRAPH_MIN.

Reply:
Indeed, I am! You're sharp. I think I'm also missing a + 1 if I want to make the max inclusive (which I did want to):
(BODY_PARAGRAPH_MAX - BODY_PARAGRAPH_MIN + 1)